raiden/ui/cli.py

```diff
@@ -505,7 +505,7 @@ def options(func: Callable) -> Callable:
         ),
     ]
 
-    if importlib.util.find_spec("IPython"):
+    if find_spec("IPython"):
```

Comment:
I prefer the qualified name for such rare functions, but it's fine either way.

Reply:
Somehow mypy does not want to accept the qualified name, even with a `# type: ignore`.
Maybe some "strict" mypy setting?
I would prefer it this way, so that mypy is happy and our pre-commit hooks don't fail constantly because of this.

Reply:
That's a strange kind of error. But this is totally unimportant, so let's ignore it.